### DISCUSSION TITLE
How to change the "/spoke" path to "/example" of url on custom hubs cloud instance .

### DISCUSSION THREAD

**Discussion by RathodDeven** [mozilla/hubs#4881] [Hubs Cloud Questions] [answered]
I have successfully deployed hubs cloud on my custom domain using aws and have also deployed custom spoke client using deploy.js script [Pull request](https://github.com/MozillaReality/ieeevr-spoke/pull/1). 
Currently, I can access spoke using https://mydomain.com/spoke but I want to change it and have spoke access using https://mydomain.com/example. 
So that when i open https://mydomain.com/example it would open up the spoke. 

What and where are the changes required to be made for changing /spoke to /example in url ?

**Comment by RathodDeven** [comment]
<img width="416" alt="image" src="https://user-images.githubusercontent.com/69627223/144019726-4b35ec72-bde5-479b-84b5-099676bb9cfd.png">
If I change ROUTER_BASE_PATH from "/spoke" to "/example" and then on https://mydomain.com/example it shows `invalid url` and on https://mydomain.com/spoke it shows `Page Not Found`

**Comment by pattersonbl2** [comment, marked as answer]
This would be a custom client setup. Currently, We are limited in our troubleshooting with custom clients because of our team size. You could ask this question in our community discord.
